docs/customizing_deis/controller_settings.rst

```diff
@@ -73,3 +74,38 @@ Be sure that your custom image functions in the same way as the `stock controlle
 Deis. Specifically, ensure that it sets and reads appropriate etcd keys.
 
 .. _`stock controller image`: https://github.com/deis/deis/tree/master/controller
+
+Unit hostname
+-------------
+Per default, Docker automatically generates a hostname for your application unit, such as: 
+``5c149b397cd6``. Auto generated hostnames is not always preferred. For instance, 
+New Relic would classify each Docker container as an unique server since they use hostname 
+for grouping applications running on the same server together.
+
+Deis supports configuring hostname assignment through the ``unitHostname`` setting.
+You can change the assignment solution using the following command:
+
+.. code-block:: console
+
+    $ deisctl config controller set unitHostname=application
+
+The valid ``unitHostname`` alternatives are:
+
+default
```

Comment:
Could we please make this a [definition list](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#definition-lists) instead?